.github/workflows/build-documentation-staging.yml

```
@@ -0,0 +1,46 @@
+name: Build documentation
+
+on: push
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Upgrade pip
+        run: |
+          # install pip=>20.1 to use "pip cache dir"
+          python3 -m pip install --upgrade pip
+      - name: Get pip cache dir
+        id: pip-cache
+        run: echo "::set-output name=dir::$(pip cache dir)"
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.pip-cache.outputs.dir }}
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: python3 -m pip install -r ./requirements.txt
+
+      - run: sphinx-multiversion docs build/html
+
+      - run: cp ./index.html ./build/html
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_branch: asf-staging
+          publish_dir: ./build/html
+          destination_dir: docs
```

```suggestion
          destination_dir: age-manual
```
If I'm not mistaken, the documentation will be in the `age-manual` folder on the new website. Thinking of this: I hope this doesn't break any links from external sites and issues.

---

I made it docs so it wouldn't overwrite the documentation that was there, for now. But we can change that.